bin/patchComponent.sh

```diff
@@ -6,21 +6,23 @@ usage()
     echo -ne "and to decrease the development && testing turnaround time.\n"
     echo -ne "Usage: \n"
     echo -ne "\t git diff --no-color | sudo ./patchComponent.sh (wmagent | reqmon | reqmgr2 | reqmgr2ms...)\n or:\n"
-    echo -ne "\t curl https://patch-diff.githubusercontent.com/raw/dmwm/deployment/pull/740.patch  | sudo ./patchComponent.sh (wmagent | reqmon | reqmgr2 | reqmgr2ms...)\n"
+    echo -ne "\t curl https://patch-diff.githubusercontent.com/raw/dmwm/WMCore/pull/11270.patch | sudo ./patchComponent.sh (wmagent | reqmon | reqmgr2 | reqmgr2ms...)\n"
     exit 1
 }
 
 # TODO: To check against a list of components
 component=$1
 [[ -z $component ]] && usage
 
+pythonVer="3.8"
```

Comment:
@todor-ivanov , how about to make it more flexible here. Just read from env and put default value, e.g. 
```
pythonVer=${PYTHON_VERSION:-3.8}
```
This will make this script more flexible and allow you to define python version in environment, which may be useful in k8s, etc. This will also allow to change to new python version easily if it will be required.

Reply:
Thanks @vkuznet that's  an excellent idea.